content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -0,0 +1,133 @@
+
+A common way to participate in the Kubernetes contributor community is
+to be everywhere. 
+
+We have an active Slack, many mailing lists, Twitter account(s), and
+dozens of community-driven podcasts and newsletters that highlight all
+end-user, contributor, and ecosystem topics. And guess what else - an
+entire repository of amazing documentation that I bet there is one gem
+you didn’t know we had! All of this information is incredibly valuable,
+but it can be too much. 
+
+Keeping up with thousands of contributors can be a challenge for anyone,
+but this task of consuming information straight from the firehose is
+particularly challenging for new community members. It’s no secret that
+the project is vast for contributors and users alike. 
+
+To paint a picture with numbers: 
+
+- 43,000 contributors
+- 6,579 members in #kubernetes-dev slack channel
+- 52 mailing lists (kubernetes-dev@ has thousands of members; sig-networking@ has 1000 alone)
+- 40 [community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups)
+- 30 [upstream meetings](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America%2FLos_Angeles) *this* week alone
+
+The scale is growing, and with that comes the need for attention.
+
+## How we got here
+
+Kubernetes (k8s for short) communication grew out of a need for people
+to connect in our growing community. With the best of intentions, the
+community spun up channels for people to connect. This energy was part
+of what helped Kubernetes grow so fast, and it also had us in a space of
+sprawling out far and wide. As adoption grew, [contributors knew there was a need for standardization](https://github.com/kubernetes/community/issues/2466).
+
+This new attention to how the community communicates led to a discovery
+of a complex web of options. There were so many options, it was a
+challenge for anyone to be sure they were in the right place to receive
+the right information. We started taking immediate action by pruning the
+inactive mailing lists and other platforms, combining where we could,
+and thinking about the future. We also asked for feedback from all our
+contributors directly [**via annual surveys**](https://github.com/kubernetes/community/tree/master/sig-contributor-experience/surveys)
+to see where folks were actually reading the news that influences their
+experiences here in our community.
+
+![Kubernetes channel access](https://user-images.githubusercontent.com/1744971/79478603-3a3a1980-7fd1-11ea-8b7a-d36aac7a097b.png)
+
+Our contributor base is larger than many enterprise companies. You can
+only imagine what it’s like getting important messages across to make
+sure they are landing and folks are taking action. 
+
+## Contributing to better communication
+
+Think about how your company/employer solves for this. Many have done so
```

Comment:
nit: Think about how your company/employer solves for this
What is solved?

Reply:
Think about how your company or employer would tackle this? 
Solving: a communications strategy, promotions, creative, etc.

Reply:
Think about how your company or employer solves communications [challenges, overflow?].
Companies build internal marketing and ... departments.

Reply:
Suggesting "Think about how your company/employer solves for this kind of communication challenge"